### PR TITLE
feat(#major); The Graph; Add The Graph Subgraph

### PR DESCRIPTION
**Pull request by steegecs**
**What has been done:**
- Set up the subgraph and made compatible with CI/CD.
- Started indexing rewards, usage metrics, and TVL from events from the Staking, Curation, and RewardsManager contracts

**To Do:**
- [x] Make sure that rewards, usage metrics, and TVL are not being double counted anywhere. 
- [x] Determine if all reward streams are present. 
- [x] Decide whether to include pools for the different roles. If any, pools for delegators and curators might make sense.
- [x] Add README doc for methodology

### CODE REVIEW

**Comment by this-username-is-taken** [comment]
For todos, check out: https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists

File: subgraphs/the-graph/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Uniswap Forks
```
Comment (this-username-is-taken) [inline subgraphs/the-graph/README.md:1]: You should probably update this doc. Specifically, include the methodology on how we are defining and calculating TVL/Revenue etc

Comment (this-username-is-taken) [reply subgraphs/the-graph/README.md:1]: Also maybe get some feedback on the methodology with protocol specialists to make sure we are all aligned

Comment (steegecs) [reply subgraphs/the-graph/README.md]: README added and ready for review.

**Review by this-username-is-taken — APPROVED** [review]
Great work! 🚀🚀🚀

**Pull request merged**